src/main/java/org/terasology/signalling/components/SignalProducerModifiedComponent.java

```diff
@@ -19,7 +19,7 @@
 import org.terasology.world.block.ForceBlockActive;
 
 /**
- * @author Marcin Sciesinski <marcins78@gmail.com>
+ *
```

Comment:
Whenever you come across an empty block (or make one by removing an `@author` tag) you can just remove the whole thing. That makes it easier for code style scanners to complain about "Hey there's no javadoc here! Add some!" :-)